clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -3161,7 +3164,8 @@ check_kernel_syscall_trace(void)
     }
     {
         std::vector<memref_with_IR_t> memref_setup = {
-            { gen_marker(TID_A, TRACE_MARKER_TYPE_FILETYPE, FILE_TYPE), nullptr },
+            { gen_marker(TID_A, TRACE_MARKER_TYPE_FILETYPE, FILE_TYPE_FULL_SYSCALL_TRACE),
```

Comment:
Trying to understand why this was changed: b/c there are no loads/stores so you can claim it has addresses?  Ditto below.

Reply:
The "Mismatching syscall num in trace start and syscall marker" test is disabled for `FILE_TYPE` (see new test below). I still wanted to test that invariant check; it is still enabled for FILE_TYPE_FULL_SYSCALL_TRACE so I modified this test to use that.